pkgs/rk3588-firmware/default.nix

```diff
@@ -0,0 +1,28 @@
+{ pkgs, stdenv, ... }:
```

Comment:
maybe set these with the defaults like the rest of the packages

Reply:
Will do!